domains/Rbfl.localplayer.dev.json

```diff
@@ -0,0 +1,18 @@
+{
+    "description": "The official website of the team RBFL",
+    "domain": ".localplayer.dev",
+    "subdomain": "RBFL",
+
+    "owner": {
+        "repo": "https://github.com/WasiTC/Rbfl",
+        "email": "wasihossain692@gmail.com"
+    },
+
+    "record": {
+        "A": ["59.152.96.161"],
+        "TXT": ["google-site-verification=gWggOMxPBF8R9hohDmlgbvq5yH8Yh_jGD5mtxAM28oY"],
+        "MX": ["josephine.ns.cloudflare.com", "toby.ns.cloudflare.com"]
```

Comment:
These MX records are Cloudflare NS records. They are not valid.

Reply:
Ok so i will remove them?

Reply:
It says josephine.ns.cloudflare.com and toby.ns.cloudflare.com are my assigned ones

Reply:
I just made a new patch containing a fix

Reply:
I turned the mx to ns

Reply:
I get it and i changed from mx to ns srry